maxim/evaluators/base_evaluator.py

```
@@ -12,34 +12,40 @@
 
 
 class BaseEvaluator(ABC):
+    """Base class for all evaluators."""
     _evaluator_names: list[str]
     _pass_fail_criteria: dict[str, PassFailCriteria]
 
     def __init__(self, pass_fail_criteria: dict[str, PassFailCriteria]):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)